authors/apps/authentication/urls.py

```diff
@@ -1,11 +1,12 @@
-from django.conf.urls import url
-
+from django.urls import path
 from .views import (
     LoginAPIView, RegistrationAPIView, UserRetrieveUpdateAPIView
 )
 
+app_name = "authentication"
+
 urlpatterns = [
-    url(r'^user/?$', UserRetrieveUpdateAPIView.as_view()),
```

Comment:
These changes seem to be unnecessarily out of the scope of your chore.

Reply:
I needed to name the urls that I specify in the tests but i can revert the changes to how the file was initially.

Reply:
I am liasing with @micahoriaso to be the on to make those changes. I am reverting to how the file was initially.